Source/Core/VideoBackends/CMakeLists.txt

```diff
@@ -1,7 +1,12 @@
 add_subdirectory(OGL)
 add_subdirectory(Null)
 add_subdirectory(Software)
+
+if(CMAKE_SYSTEM_NAME STREQUAL "Windows")
```

Comment:
All checks should be changed to this. It's currently mixed in the sources already